xde/src/xde.rs

```diff
@@ -98,7 +97,6 @@ use oxide_vpc::engine::VpcParser;
 //
 // Unwrap: We know all of these are safe to unwrap().
 const FW_FT_LIMIT: Option<NonZeroU32> = NonZeroU32::new(8096);
```

Comment:
Instead of having these options that we need to unwrap later, we can just use `NonZeroU32::new_unchecked`. While the name says unchecked, if the value is a constant, it _is_ actually checked at compile time.

https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=9a0bcc48ceff704e8d08476235325842

Reply:
Nice find, thanks! I'll use that.